omod/src/main/java/org/openmrs/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImpl.java

```diff
@@ -68,7 +70,11 @@
             }
             int formVersion = isNotEmpty(version) ? Integer.parseInt(version) : 0;
             if (formVersion > 0 && isNotEmpty(refVersion)) {
-                JSONObject previousTranslationsJson = getTranslations(new File(getFileName(formName, refVersion)));
+                File refTranslationFile = new File(getFileName(formName, refVersion));
+                if(!refTranslationFile.exists()){
+                    refTranslationFile = new File(getFileName(normalizedFileName, refVersion));
```

Comment:
why can't we pass  `normalizedFileName` directly to get the refTranslationFile ?? is the exists check required ??

Reply:
That is for backward compatability. For ex:- if we republish the existing form it will first check the translation file with old name else we will get with normalized name